bridges/TorrentGalaxyBridge.php

```diff
@@ -0,0 +1,121 @@
+<?php
+
+class TorrentGalaxyBridge extends BridgeAbstract {
+
+	const NAME = 'Torrent Galaxy Bridge';
+	const URI = 'https://torrentgalaxy.to';
+	const DESCRIPTION = 'Returns latest torrents';
+	const MAINTAINER = 'GregThib';
+	const CACHE_TIMEOUT = 14400; // 24h = 86400s
+
+	const PARAMETERS = array(
+		array(
+			'search' => array(
+				'name' => 'search',
+				'required' => true,
+				'title' => 'Type your query'
+			),
+			'lang' => array(
+				'name' => 'language',
+				'type' => 'list',
+				'exampleValue' => 'All languages',
+				'title' => 'Select your language',
+				'values' => array(
+					'All languages' => '0',
+					'English' => '1',
+					'French' => '2',
+					'German' => '3',
+					'Italian' => '4',
+					'Japanese' => '5',
+					'Spanish' => '6',
+					'Russian' => '7',
+					'Hindi' => '8',
+					'Other / Multiple' => '9',
+					'Korean' => '10',
+					'Danish' => '11',
+					'Norwegian' => '12',
+					'Dutch' => '13',
+					'Manderin' => '14',
+					'Portuguese' => '15',
+					'Bengali' => '16',
+					'Polish' => '17',
+					'Turkish' => '18',
+					'Telugu' => '19',
+					'Urdu' => '20',
+					'Arabic' => '21',
+					'Swedish' => '22',
+					'Romanian' => '23'
+				)
+			)
+		)
+	);
+
+	public function collectData(){
+		$url = self::URI . '/torrents.php' .
+			'?search=' . urlencode($this->getInput('search')) .
+			'&lang=' . $this->getInput('lang') .
+			'$sort=id&order=desc';
```

Comment:
For example, Travis currently complains about this statement. You'll know why if you remove the newlines:

```php
$url = self::URI . '/torrents.php' . '?search=' . urlencode($this->getInput('search')) . '&lang=' . $this->getInput('lang') . '$sort=id&order=desc';
```

this should be

```php
$url = self::URI . '/torrents.php?search=' . urlencode($this->getInput('search')) . '&lang=' . $this->getInput('lang') . '$sort=id&order=desc';
```

or with newlines

```php
$url = self::URI
. '/torrents.php?search=' . urlencode($this->getInput('search'))
. '&lang=' . $this->getInput('lang')
. '$sort=id&order=desc';
```